lib/src/serializers/json_api.dart

```diff
@@ -19,12 +19,15 @@ class JsonApiSerializer implements Serializer {
   @override
   JsonApiManyDocument deserializeMany(String payload) {
     Map<String, dynamic> parsed = parse(payload);
-    var docs = (parsed['data'] as Iterable).map((item) => JsonApiDocument(
-        item['id'],
-        item['type'],
-        item['attributes'],
-        item['relationships'],
-        parsed['included']));
+    var docs = (parsed['data'] as List)
+        .map((item) => JsonApiDocument(
+              item['id'],
+              item['type'],
+              item['attributes'],
+              item['relationships'],
+              parsed['included'],
+            ))
+        .toList();
     return JsonApiManyDocument(docs, parsed['included'], parsed['meta']);
```

Comment:
@halil81 We can't be sure that `parsed['included']` will return a `List`, so let's use `List.from()`

Reply:
Thanks! @algodave, `parsed['included']` wrapped in `List.from()`